README.md

```diff
@@ -35,7 +35,14 @@ The full documentation of the latest release of Cake for Rider can be found at [
 
 ### Documentation changes in the upcoming release
 
-None.
+#### Cake .NET Tool install option
+It is now possible to install (or update to) the current version of the Cake .NET Tool:
```

Comment:
It might make sense to add "(Global)" after "Cake .NET Tool" to match what the VS Code extension uses:

![image](https://user-images.githubusercontent.com/177608/106084458-c64aa700-60f4-11eb-9950-baa1c718e4fc.png)

Reply:
changed.